scripts/eosio_build_darwin.sh

```diff
@@ -211,170 +214,302 @@
 				esac
 			done
 		fi
-		printf "\tInstalling boost libraries.\n"
+		printf "\\tInstalling boost libraries.\\n"
 		if [ ! -d "/usr/local/Cellar/boost/1.66.0" ]; then
 			if ! brew install "https://raw.githubusercontent.com/Homebrew/homebrew-core/76a83d798d76aa43459ceb145b721ba4a16a54a6/Formula/boost.rb"
 			then
-				printf "\tUnable to install boost 1.66 libraries at this time. 0\n"
-				printf "\tExiting now.\n\n"
-				exit;
+				printf "\\tUnable to install boost 1.66 libraries at this time. 0\\n"
+				printf "\\tExiting now.\\n\\n"
+				exit 1;
 			fi
-			printf "\tPinning boost libraries.\n"
+			printf "\\tPinning boost libraries.\\n"
 			if ! brew pin boost
 			then
-				printf "\tUnable to pin boost libraries at this time. 2\n"
-				printf "\tExiting now.\n\n"
-				exit;
+				printf "\\tUnable to pin boost libraries at this time. 2\\n"
+				printf "\\tExiting now.\\n\\n"
+				exit 1;
 			fi
 		else
 			if ! brew switch boost 1.66.0
 			then
-				printf "\tUnable to swtich boost libraries to 1.66.0 at this time. 1\n"
-				printf "\tExiting now.\n\n"
+				printf "\\tUnable to swtich boost libraries to 1.66.0 at this time. 1\\n"
+				printf "\\tExiting now.\\n\\n"
 				exit;
 			fi
 		fi
-		printf "\tBoost 1.66.0 successfully installed @ /usr/local.\n"
+		printf "\\tBoost 1.66.0 successfully installed @ /usr/local.\\n"
 	else
-		printf "\tBoost 1.66.0 found at /usr/local.\n"
+		printf "\\tBoost 1.66.0 found at /usr/local.\\n"
 	fi
 
-	printf "\n\tChecking MongoDB C++ driver installation.\n"
-    if [ ! -e /usr/local/lib/libmongocxx-static.a ]; then
-		cd ${TEMP_DIR}
-		brew install --force pkgconfig
-		brew unlink pkgconfig && brew link --force pkgconfig
+	printf "\\n\\tChecking MongoDB C++ driver installation.\\n"
+    if [ ! -e "/usr/local/lib/libmongocxx-static.a" ]; then
```

Comment:
Note the slim branch does not have a mongodb plugin and will likely not have one for June.

Reply:
I have $( git branch | grep \* ) writing the current branch to stdout.
I can add a check in to install mongo when necessary. We need to define what that condition/branch is.

Reply:
It will be master branch soon.